tracing-subscriber/src/fmt/format.rs

```diff
@@ -731,6 +806,21 @@ mod test {
         assert_eq!(expected, actual.as_str());
     }
 
+    #[cfg(feature = "json")]
+    #[test]
+    fn json() {
+        lazy_static! {
+            static ref BUF: Mutex<Vec<u8>> = Mutex::new(vec![]);
+        }
+
+        let make_writer = || MockWriter::new(&BUF);
+
+        let expected =
+            "{\"timestamp\":\"fake time\",\"level\":\"INFO\",\"target\":\"tracing_subscriber::fmt::format::test\",\"message\":\"some json test\"}\n";
```

Comment:
Should we also have a test for serializing an event inside of a span?

Reply:
Yes. And we should test compact events too with the non-json formatters...